src/RoslynPad.Common.UI/ViewModels/MainViewModelBase.cs

```diff
@@ -106,7 +106,7 @@ public MainViewModelBase(IServiceProvider serviceProvider, ITelemetryProvider te
 
         private async Task InitializeInternal()
         {
-            RoslynHost = await Task.Run(() => new RoslynHost(NuGetConfiguration, CompositionAssemblies, 
+            RoslynHost = await Task.Run(() => new RoslynHost(NuGetConfiguration, CompositionAssemblies,
```

Comment:
Nit: Please be mindful of formatting changes in the future. They tend to make PR reviews harder.

Reply:
yea, sorry about that, habit of ctrl+k,d